### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by millette** [mafintosh/webassembly-binary-toolkit#1]
It got to 100% on Debian (Jessie).

```
[100%] Building CXX object CMakeFiles/wasm-shell.dir/src/wasm-interpreter.cpp.o
Linking CXX executable bin/wasm-shell
[100%] Built target wasm-shell
```

But then failed, and ```wast2wasm``` isn't found in my path.

Errors from npm-debug.log below (I can provide a full log if required).

```
149 verbose stack Error: ENOENT: no such file or directory, chmod '/home/millette/n/lib/node_modules/webassembly-binary-toolkit/wabt/out/clang/Debug/wast2wasm'
149 verbose stack     at Error (native)
150 verbose cwd /home/millette
151 error Linux 3.16.0-4-amd64
152 error argv "/home/millette/n/bin/node" "/home/millette/n/bin/npm" "install" "-g" "webassembly-binary-toolkit"
153 error node v6.11.0
154 error npm  v3.10.10
155 error path /home/millette/n/lib/node_modules/webassembly-binary-toolkit/wabt/out/clang/Debug/wast2wasm
156 error code ENOENT
157 error errno -2
158 error syscall chmod
159 error enoent ENOENT: no such file or directory, chmod '/home/millette/n/lib/node_modules/webassembly-binary-toolkit/wabt/out/clang/Debug/wast2wasm'
160 error enoent ENOENT: no such file or directory, chmod '/home/millette/n/lib/node_modules/webassembly-binary-toolkit/wabt/out/clang/Debug/wast2wasm'
160 error enoent This is most likely not a problem with npm itself
160 error enoent and is related to npm not being able to find a file.
161 verbose exit [ -2, true ]
```

**Comment by mafintosh** [comment]
fixed in 1.0.2

**Comment by leyayun** [comment]
@mafintosh I have the same issue on OSX 10.13.5 with webassembly-binary-toolkit@2.0.0
